registry.json

```diff
@@ -26,6 +26,33 @@
             "downloadURL": "https://github.com/SparedRay/superpowers-game-SparedRay-easystarjs-plugin/releases/download/v0.1.0/superpowers-easystarjs-api.zip",
             "releaseNotesURL": "https://github.com/SparedRay/superpowers-game-SparedRay-easystarjs-plugin/releases"
           }
+        },
+        "florentpoujol": {
+          "fText": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-ftext-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-ftext-plugin/releases"
+          },
+          "fMouseInput": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-fmouseinput-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-fmouseinput-plugin/releases"
+          },
+          "fLang": {
+            "version": "0.4.1",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-flang-plugin/archive/v0.4.1.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-flang-plugin/releases"
+          },
+          "DOM": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-dom-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-dom-plugin/releases"
+          },
+          "THREE.js": {
```

Comment:
@bilou84 do you know if this will work? Folders with dots in their name are ignored but I'm not sure if this is used as the folder name or just as a label?